docs/gds-supported-platform/permissions.md

```diff
@@ -0,0 +1,52 @@
+# Permissions
+
+The permissions a given user will have in a given namespace in a given cluster
+depend on their configuration in [gds-trusted-developers]. Specifically, the
+`roles` they have.
+
+## Cluster admins
+
+A `gds-trusted-developer` may be configured as a cluster admin in all clusters
```

Comment:
can we have a note that the admin is role is intended to only be granted for shorter periods of time and to a small group of users, such as during incidents or support

Reply:
I've updated the content a bit. Could you take another look?